habitat_sim/utils/datasets_download.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import os
+import shutil
+import zipfile
+
+import git
+
+repo = git.Repo(".", search_parent_directories=True)
```

Comment:
Meta Point: if we want this to live in habitat-sim/utils the git repo trick will break for a global/conda install, no? Also putting it in the global namespace liek this will prevent this code from being imported or run outside of a git repo even if that path is specified, which is less than ideal. 

Should all these print statements use the habitat_sim logger?